pkg/asset/manifests/openstack/cloudproviderconfig.go

```diff
@@ -65,9 +69,10 @@ func CloudProviderConfigSecret(cloud *clientconfig.Cloud) ([]byte, error) {
 	if cloud.RegionName != "" {
 		res.WriteString("region = " + strconv.Quote(cloud.RegionName) + "\n")
 	}
-	if cloud.CACertFile != "" {
```

Comment:
How about leaving `ca-file` in the `clouds.yaml` but change the path to a well known location like you're doing with `clouds.conf`? The pod would need to mount the configmap containing the cert to the agreed path -- just in case something read `ca-file` from `clouds.yaml`.

Reply:
Its actually a bug. Passing in a ca-file in the clouds.yaml is broken right now. But yes, I should still pass the same known path here.